src/_nebari/stages/kubernetes_services/template/modules/kubernetes/forwardauth/main.tf

```diff
@@ -144,7 +144,7 @@ resource "kubernetes_manifest" "forwardauth-middleware" {
     apiVersion = "traefik.containo.us/v1alpha1"
     kind       = "Middleware"
     metadata = {
-      name      = "traefik-forward-auth"
+      name      = var.forwardauth_middleware_name
```

Comment:
why was there a change here?

Reply:
We previously hard coded the forward auth middleware name.  I added a variable for it.  I also created an output for whatever name is chosen.  I needed it for an mlflow plugin.  See https://github.com/Quansight/nebari-mlflow-plugin/blob/0eeb522380e635ac4c5ac6d3ba843427acc6923f/src/nebari_plugin_mlflow_aws/__init__.py#L207-L221

Reply:
cool! The reason I asked was because DaskGateway also uses it, so I just wanted to make sure that the service would still work